liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/StartDebugMojoSupport.java

```diff
@@ -38,54 +39,38 @@
 
     /**
      * Location of customized configuration directory
-     * 
-     * @parameter expression="${configDirectory}"
      */
+    @Parameter(property = "configDirectory")
     protected File configDirectory;
 
     /**
      * Location of customized configuration file server.xml
-     * 
-     * @parameter expression="${configFile}"
-     *            default-value="${basedir}/src/test/resources/server.xml"
      */
+    @Parameter(property = "configFile",defaultValue = "${basedir}/src/test/resources/server.xml")
     protected File configFile;
 
     /**
-     * 
      * Location of bootstrap.properties file.
-     * 
-     * @parameter expression="${bootstrapPropertiesFile}"
-     *            default-value="${basedir}/src/test/resources/bootstrap.properties"
      */
+    @Parameter(property = "bootstrapPropertiesFile",defaultValue = "${basedir}/src/test/resources/bootstrap.properties")
     protected File bootstrapPropertiesFile;
 
-    /**
-     * @parameter
-     */
+    @Parameter(property = "bootstrapProperties")
```

Comment:
Same comment as I had, do you know how to use -D to provide value for this property.